cubedash/index/postgis/_api.py

```diff
@@ -376,39 +355,65 @@ def collections_search_query(
         bbox: tuple[float, float, float, float] | None = None,
         time: tuple[datetime, datetime] | None = None,
         q: Sequence[str] | None = None,
-    ) -> Result:
-        collection = self.collection_cols().subquery()
-        query = select(collection).where(collection.c.period_type == "all")
-
+    ) -> list[Row]:
+        # STAC Collections only hold a bounding box in EPSG:4326, no geometry
+        # Calculate the bounding box on the server, it's far more efficient
+        collection_bbox = func.Box2D(
```

Comment:
The RHS takes its data from `TimeOverview`, shouldn't this be called `t_o_bbox` (or `overview_bbox`, but that is longer)?

Reply:
Hmmm. I might add some comments.

The Cubedash Product (which maps to a STAC Collection) doesn't have any bounding box or geometry attached, all the geometries are in the TimeOverview table, grouped by different `period_types`. In this case we use the `period_type=="all"` to get the one that covers all of time.